CMakeLists.txt

```diff
@@ -21,9 +21,9 @@ cmake_minimum_required(VERSION 3.16)
 set(GROONGA_NORMALIZER_MYSQL_PROJECT_NAME "groonga-normalizer-mysql")
 project("${GROONGA_NORMALIZER_MYSQL_PROJECT_NAME}")
 
-if(NOT DEFINED GROONGA_NORMALIZER_MYSQL_DATA_DIR)
-  set(GROONGA_NORMALIZER_MYSQL_DATA_DIR
-    "share/${GROONGA_NORMALIZER_MYSQL_PROJECT_NAME}")
+if(NOT DEFINED GROONGA_NORMALIZER_MYSQL_DOC_DIR)
+  set(GROONGA_NORMALIZER_MYSQL_DOC_DIR
+    "share/doc/${GROONGA_NORMALIZER_MYSQL_PROJECT_NAME}")
```

Comment:
We should use `CMAKE_INSTALL_DOCDIR` by including `GNUInstallDirs`:

* https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html
* https://github.com/groonga/groonga/blob/8856bb16ca48d3c936eb36e83fd269835ab055d2/CMakeLists.txt#L143